modules/diskq/tests/test_diskq_truncate.c

```diff
@@ -216,7 +216,8 @@ Test(diskq_truncate, test_diskq_truncate_on_push)
   GString *filename = g_string_new("test_dq_truncate_on_write.rqf");
 
   DiskQueueOptions options;
-  q = _create_reliable_diskqueue(filename->str, &options);
+  q = _create_reliable_diskqueue(filename->str, &options, FALSE);
+  options.truncate_size_ratio = 0.01;
```

Comment:
Is it safe to set a Qdisk option after load-queue?

Reply:
By the way, why set it if it's the same as the default value?

Reply:
I changed the order of the setting, and obtained the default value from `DiskQueueConfig`, so these should be fixed now.